libraries/lib-ipc/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+#[[
+]]#
```

Comment:
Second '#' isn't needed.

Add a few words about the general purpos of these files.